tools/devtools/src/devtools/tabs/History.tsx

```diff
@@ -180,6 +191,9 @@ export function History({
                 >
                   ⇥
                 </button>
+                <button onClick={toggleHidePresenceEvent}>
```

Comment:
While it would be nice to express it as a tooltip, how about using title to indicate the role of the button for now?
In the future, we might want to consider upgrading this to a more robust tooltip solution for enhanced user experience.
```jsx
 <button
    className="toggle-tab-btn"
    title={ hidePresenceEvents ? 'Show all changes' : 'Show only root changes' }
```